index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+<body>
+    
+    <header>
+        <div class="logo">Vishal</div>
+       
+        <div class="burger">
+            <i class="fa fa-bars fa-2x" aria-hidden="true"></i>
+        </div>
```

Comment:
- [ ] I will strongly recommend that you wrap your hamburger in a `nav` tag because the hamburger will contain the navigation elements which you will be adding later on and the `nav` element is a semantic element that is used to contain navigation elements. So to improve your website accessibility, it will be important to use the nav element 👍🏽 .

e.g.

  
```
  <nav class="burger">
      <i class="fa fa-bars fa-2x" aria-hidden="true"></i>
  </nav>
```